test/threads_test.py

```diff
@@ -79,7 +96,7 @@ def test_wait(self):
 
 
 class ThreadsModuleTest(unittest.TestCase):
-    def todo_test_benchmark_workers(self):
+    def test_benchmark_workers(self):
         "tags:long_running"
```

Comment:
need a little docstring here for the test. Just something like:

```
""Ensure benchmark_workers performance measure is functional""
```

Keeps the linters happy.

Reply:
Added